README.md

```diff
@@ -728,6 +728,25 @@ Using transports in the same process causes unnecessary load and slows down Node
 
 If you write a transport, let us know and we will add a link here!
 
+<a name="pino-elasticsearch"></a>
+### pino-elasticsearch
+
+[pino-elasticsearch][pino-elasticsearch] uploads the log lines in bulk
+to [ElasticSearch][elasticsearch], to be displayed in [Kibana][kibana].
+
+It is extremely simple to use and setup
+
+```sh
+$ node yourapp.js | pino-elasticsearch --host 192.168.1.42
+```
+
+Assuming ElasticSearch is running on 192.168.1.42. Then, head to your
+Kibana instance, and create an index pattern
+(https://www.elastic.co/guide/en/kibana/current/setup.html) on `pino`,
+the default for `pino-elasticsearch`.
```

Comment:
Which ever way is the most basic and compatible with kibana - i'd put it in quote marks instead of back ticks though because backticks make it look like we're talking about a pino instance rather than a field value on kibana setup i.e.

'pino'

or maybe `'pino'`